cmds/webboot/types.go

```diff
@@ -12,18 +12,19 @@ var bookmarks = map[string]string{
 }
 
 // ISO contains information of the iso user want to boot
+// var _ = Entry(&ISO{})
```

Comment:
Don't leave this commented out, right?  It's a variable declaration. It does not work if it is commented out.

Reply:
oh so you mean just let it be part of the code. just define the var but not use? Interesting! I haven't imaged that it could be like this

Reply:
yes, it's a variable declaration.
var _ = Entry(&ISO{})